reference_conf/README.md

```diff
@@ -0,0 +1,12 @@
+Reference configuration
+-----------------------
```

Comment:
This is reStructuredText, I had to resort to that since markdown doesn't support file inclusions.
As explained in my `docs.yaml` comment, this needs to use the extension `.md` right now (probably a bug), which means it's not rendered correctly on GitHub.

I hope the include will work on GitHub once we use the right extension, but at worst it's not hard to figure it out.